File: data/museum/oseg_generated/ruby/ListSpecialEvents_default_example.rb
require "openapimuseum_client"

OpenapiMuseumClient.configure do |config|
end

begin
    api_caller = OpenapiMuseumClient::EventsApi.new

    response = api_caller.list_special_events(
        {
            start_date: "2023-02-23",
            end_date: "2023-04-18",
            page: 2,
            limit: 15,
        },
    )

    p response
rescue OpenapiMuseumClient::ApiError => e
    puts "Exception when calling Events#list_special_events: #{e}"
end

File: data/museum/oseg_generated/csharp/ListSpecialEvents_default_example.cs
using System;
using System.Collections.Generic;
using System.IO;

using Org.OpenAPIMuseum.Api;
using Org.OpenAPIMuseum.Client;
using Org.OpenAPIMuseum.Model;

public class ListSpecialEventsDefaultExample
{
    public static void Main()
    {
        var config = new Configuration();

        try
        {
            var apiCaller = new EventsApi(config);

            var response = apiCaller.ListSpecialEvents(
                startDate: "2023-02-23",
                endDate: "2023-04-18",
                page: 2,
                limit: 15
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling Events#ListSpecialEvents: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}
